java/hsfs/src/main/java/com/logicalclocks/hsfs/FeatureViewBase.java

```diff
@@ -323,95 +321,6 @@ public List<List<Object>> getFeatureVectors(Map<String, List<Object>> entry, boo
     return vectorServer.getFeatureVectors(this, entry, external);
   }
 
-  /**
-   * Add name/value tag to the feature view.
-   *
-   * @param name
-   *     name of the tag
-   * @param value
-   *     value of the tag. The value of a tag can be any valid json - primitives, arrays or json objects
-   * @throws FeatureStoreException
-   * @throws IOException
-   */
-  public abstract void addTag(String name, Object value) throws FeatureStoreException, IOException;
```

Comment:
why removing all the abstract method? Are they not needed for spark/flink etc? same for `featurestorebase` and `featuregroupbase`.
I think you can implement the abstract method as "not available" in the `FeatureView`

Reply:
My point was that  who will use `addTag` from Flink or Beam? It doesn't make sense. But it is possible do add since only api classes are used